src/mobile/src/ui/views/onboarding/LanguageSetup.js

```diff
@@ -115,7 +114,7 @@ class LanguageSetup extends Component {
         return (
             <TouchableWithoutFeedback
                 onPress={() => {
-                    if (this.dropdown) {
+                    if (isIOS) {
```

Comment:
Why are we checking platform here? Do we not want to close the dropdown on android?

Reply:
Android dropdown covers the whole screen, so you can't click on the bg to close it

Reply:
You need to test it to see what I mean